### PR TITLE
Feature/relative url

### PR DESCRIPTION
**Pull request by yamajik**
Issue: #241

### CODE REVIEW

**Comment by yamajik** [comment]
Problems still exist while using relative url ("resource").
location.pathname + relative_url will fix it.

**Comment by umdstu** [comment]
@kylecarbs - I know this is only a week old, but do you have an estimate for when you'll be able to review this PR?

Thanks!

**Comment by au5ton** [comment]
polite ping

File: packages/ide/src/fill/electron.ts [hunk 1]
```diff
@@ -47,7 +47,8 @@ const newCreateElement = <K extends keyof HTMLElementTagNameMap>(tagName: K): HT
 			},
 			set: (value: string): void => {
 				if (value) {
-					value = value.replace(/file:\/\//g, "/resource");
+					const resourceBaseUrl = location.pathname.replace(/\/$/, "") + "/resource";
```
Comment (kylecarbs) [inline packages/ide/src/fill/electron.ts]: Wouldn't this only catch URLs that end with `/`?

Something like `https://ide.kwc.io/banana` wouldn't be caught with this.

Comment (yamajik) [reply packages/ide/src/fill/electron.ts]: This makes sure the url is correct.

Both 
`https://ide.kwc.io/banana`
`https://ide.kwc.io/banana/`
use
`https://ide.kwc.io/banana/resource`
to request the resources

**Comment by ShannonMasters** [comment]
any chance of getting this merged in for the next release?  this is a pretty big issue for how my environment is atm (of which I have limited control)

**Pull request merged**